artifactory/cli.go

```diff
@@ -952,6 +952,10 @@ func getSearchFlags() []cli.Flag {
 			Name:  "build",
 			Usage: "[Optional] If specified, only artifacts of the specified build are matched. The property format is build-name/build-number. If you do not specify the build number, the artifacts are filtered by the latest build number.` `",
 		},
+		cli.BoolFlag{
```

Comment:
Since this option is shared now between two commands, I suggest using a new *getIncludeDirsFlag* function.